### DISCUSSION TITLE
Bảo vệ header project nhưng vẫn lỗi

### DISCUSSION THREAD

**Discussion by luluhoward** [codingreshapefuture/Community#21] [Q&A] [answered]
Bảo vệ header project nhưng vẫn lỗi

**Comment by lucthienphong1120** [comment, marked as answer]
Nếu bạn define tieude trên const.h như vậy thì khi include const.h ở đâu, nó sẽ define lại tieude ở chỗ đó.

Cách tốt hơn thì bạn nên define tieude trong 1 file cpp vd như ở đây là const.cpp

rồi extern nó trong const.h
